src/main/java/org/example/model/ProductVersion.java

```diff
@@ -0,0 +1,33 @@
+package org.example.model;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+@Entity
+@Getter
+@Setter
+@ToString(exclude = "owner")
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "product_version")
+public class ProductVersion {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private long id;
+
+    @Column
+    private String details;
+
+    @Column
+    private String creator;
+
+    @Column
+    private int version;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "product_id", referencedColumnName = "id") //
+    private Product owner;
```

Comment:
why owner and not product? It's about name

Reply:
owner - changed to "product"